trace/trace_context.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2016, Google Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.instrumentation.trace.proto;
+
+option java_multiple_files = true;
```

Comment:
I don't know if this matters, but `stats_context.proto` uses nested classes instead of multiple files: https://github.com/bogdandrutu/instrumentation-proto/blob/575cdb257cf149a186dc2276dcdb4c3ffba9ff1a/stats/stats_context.proto#L17